### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by mattiaverga** [XEphem/XEphem#61]
Running `make run-test` I get:

```
./test > output.actual
diff -u output.expected output.actual
--- output.expected     2022-01-27 20:27:21.000000000 +0100
+++ output.actual       2022-10-06 10:37:30.364043236 +0200
@@ -16,4 +16,4 @@
 
 Pluto discontinuity when Chapront (1995) ends on 2247/10/1
 RA = 16:52:53.80
-RA = 11:52:36.32
+RA = 16:46:16.01
make: *** [Makefile:5: run-test] Errore 1
```

**Comment by brandon-rhodes** [comment]
@mattiaverga — Drat. I quickly wrote up `run-test` for my own purposes to make sure I didn't break XEphem as I merged a few pull requests, and then forgot to keep running it, and when I added the new GitHub CI steps I forgot it was there, as it's a perfect target for the CI to run on every pull request.

And so when I merged an update to Pluto's orbit, I didn't get a reminder that the test output also needed to be updated.

This week or next I'll plan to add `run-test` to the CI steps, and then fix the output.

@rmathar — Would that degrade the user experience in any way, like removing Pluto from the list of planets that folks can select from? Or would it be invisible from a user-facing features point of view?

**Comment by rmathar** [comment]
My proposal is definitely to remove, to purge, to delete, to erase Pluto from any list that starts with Mercury, Venus, Earth etc. This has the nice side effect that jumps in Pluto's orbit which are created by finite fitting functions disappear because the actively maintained ephemerides database of the MPC takes over, and it will add the user experience that after the 2006 IAU decision on nomenclature XEphem starts to conform with that. People who intend to give Pluto a special status would need to add the MPC target "134340 Pluto" to their favourites list.

**Comment by pidloop** [comment]
Greetings all,

This is a tough one. On the one hand, removing Pluto altogether would be
officially correct but even today would be jarring to many users. On the
other hand, there's no question using MPC elements is more accurate and
should be encouraged.

How about this for a really lame idea: leave everything like it is now, but
whenever the user chooses Pluto, raise a popup recommending use of MPC,
maybe even with a summary of how to do that.

I'll show myself out.

Elwood

**Comment by brandon-rhodes** [comment]
> removing Pluto altogether would be officially correct but even today would be jarring to many users…

I agree with Elwood. In addition, maybe folks' favorite files will already list `Pluto,P` and I would like those existing settings to keep working without any change. If we were writing a new planetarium program today, we could make a different call, but today the change would break workflows for longtime users. I don't think correctness in nomenclature is to be bought at the expense of our users.

> raise a popup recommending use of MPC…

For now let's try something less intrusive: an update to the docs, which I can easily add, mentioning the possibility of loading Pluto as a minor planet.

> the actively maintained ephemerides database of the MPC takes over…

We always have the option of keeping the Pluto orbit up-to-date with whatever the current MPC elements are, and of removing the step function if we think that leads to better results for users. Next week I'll try to review the history of our Pluto support, as I remember one or two bumps in the road over the past couple of years (though maybe one was simply that someone submitted a bad orbit?), and reply here again.

**Comment by brandon-rhodes** [comment]
As the failing test has been fixed in 6e454dff, I'm going to go ahead and close this issue. Also, we're keeping Pluto, because — let's face it — part of the fun of running an old 1990s X Windows astronomy program is seeing nine planets instead of eight. It feels all retro. But thanks for the comments, everyone!

(And I'm going to sit down right now and try out 134340 and see if the docs should suggest that instead, or even the UI.)
